RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,240 @@
+## Subject Line
+[Blox] Deploy release - v$$blox_version$$
+
+## Activity Details
+This activity is to: Release Blox version v$$blox_version$$ by pushing our code up to the master branch on GitHub, and pushing new Docker images up to Docker Hub.
+
+The purpose of this change is to: Release the latest features and bug fixes for Blox to allow consumers to start using the new functionality.
+
+The Blox framework version is: v$$blox_version$$ with a git hash of $$github_hash$$
+
+## Impact Details
+What will happen if this CM doesn't happen?  
+Blox consumers will not be able to take advantage of the new features and bug fixes in Blox to schedule tasks on ECS.
+
+Why is this the correct time/day to complete the CM?  
+This corresponds to the completion date of all GitHub issues planned for this release.
+
+Are there any related, prerequisite changes upon which this CM hinges?  
+No.
+
+## Worst Case Scenario
+What could happen if this change causes impact?  
+TODO: Fill this part in.
+
+Where are the most likely places this change will fail?  
+TODO: Fill this part in.
+
+## Hostname or Service
+GitHub: https://github.com/blox/blox  
+Docker Hub: https://hub.docker.com/u/bloxoss/
+
+## Timeline / Activity Plan
+Times are relative to the start of the CM.
+
+- 00:00 Change the CM status to "In Progress".
+- 00:05 Merge the GitHub pull request into the master branch.
```

Comment:
Ah I see you have it below. In that case you can just refer to it by saying "perform the activity checklist detailed below" so someone who's following these steps doesn't skip it because they haven't seen it. That can come after the in-progress step.